kokoro/linux/64-bit/Dockerfile

```diff
@@ -66,6 +66,8 @@ RUN apt-get clean && apt-get update && apt-get install -y --force-yes \
   openjdk-7-jdk \
   # -- For java_oracle7 -- \
   oracle-java7-installer \
+  # -- For java_linkage_monitor \
```

Comment:
Have you pushed this docker image to our docker hub?
https://hub.docker.com/search?q=protobuftesting&type=image